main.py

```diff
@@ -1,10 +1,52 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+        player_guild = player_data.get("guild")
+        player_race_name = player_data.get("race", {}).get("name")
+        player_race_description = \
+            player_data.get("race", {}).get("description")
+        player_skills = player_data.get("race", {}).get("skills")
+
+        guild = None
+        if player_guild is not None:
+            guild, _ = Guild.objects.get_or_create(
+                name=player_guild.get("name"),
+                defaults={
+                    "description": player_guild.get("description")
+                }
+            )
+
+        if not player_race_description:
+            player_race_description = ""
+        race, _ = Race.objects.get_or_create(
+            name=player_race_name,
+            defaults={
+                "description": player_race_description
+            }
+        )
+
+        for skill in player_skills:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                defaults={"race": race}
+            )
```

Comment:
But if one race(Elf) will have for example skill with name "Berserk" with bonus = 2
And another(Dwarf) also want to have skill with name "Berserk" and bonus = 2
You will not create skill for dwarf. And it will be incorrect

Reply:
In task it's said that "Each race has unique skills." That's why I consider that two races can not have skills with the same name.